inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -1388,6 +1388,7 @@ cdef class IENetwork:
                 self.impl = C.IENetwork(model_, weights_)
             else:
                 self.impl = C.IENetwork()
+                free(bin_buffer)
```

Comment:
cannot with use something like unique_ptr here?

Reply:
No, TBlob constructor expects bin_buffer as raw pointer and takes care of freeing it

Reply:
you can use unique_ptr::release to give ownership to TBlob, while in case of exceptions unique_ptr will safely release the memory.